src/services/credentialEvents/index.ts

```diff
@@ -115,6 +139,16 @@ export default class CredentialEvents {
       formatData.proposal?.anoncreds.schema_version === maybeSchema.version
     )
   }
+  private isCredentialError(credential: Credential): boolean {
+    if (
+      credential.role === 'holder' &&
+      credential.state === 'abandoned' &&
+      typeof credential.errorMessage === 'string'
+    ) {
+      return true
+    }
+    return false
```

Comment:
this can be simplified to just 

```
return credential.role === 'holder' &&
      credential.state === 'abandoned' &&
      typeof credential.errorMessage === 'string'
```